api/envoy/extensions/filters/listener/proxy_protocol/v3/proxy_protocol.proto

```diff
@@ -40,4 +40,9 @@ message ProxyProtocol {
 
   // The list of rules to apply to requests.
   repeated Rule rules = 1;
+
+  // NOTICE: only enable if ALL traffic to the listener comes from a trusted source.
```

Comment:
Should probably in an `.. attention::` clause.
See [example](https://github.com/envoyproxy/envoy/blob/6df4b00211a667396752e9c90d6a89860011c319/api/envoy/config/cluster/v3/cluster.proto#L775-L779). 

I also suggest to rephrase the comment, starting with with it does, its default value, and then the notice part.

Reply:
Thanks for the example, addressed in https://github.com/envoyproxy/envoy/pull/18951/commits/91c9a1b8571e6056de9292ff4470f51020213732